ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -831,6 +832,7 @@ static ProtocolSpecBuilder experimentalEipsDefinition(
                 MainnetEVMs.experimentalEips(
                     gasCalculator, chainId.orElse(BigInteger.ZERO), evmConfiguration))
         .depositsValidator(new DepositsValidator.AllowedDeposits(depositContractAddress))
+        .exitsValidator(new ValidatorExitsValidator.AllowedExits())
```

Comment:
`experimentalEipsDefinition` is built out `futureEipsDefinition` which is build out of prague. I don't think we need to add this here.